sys/mips/mips/elf_machdep.c

```diff
@@ -159,11 +159,12 @@ static __ElfN(Brandinfo) freebsd_brand_info = {
 	.emul_path	= NULL,
 	.interp_path	= "/libexec/ld-elf.so.1",
 	.sysvec		= &elf_freebsd_sysvec,
-	.interp_newpath	= NULL,
 #if __has_feature(capabilities)
+	.interp_newpath	= "/libexec/ld-cheri-elf.so.1",
 	.header_supported = mips_elf_header_supported,
```

Comment:
Since we've removed CHERI256 support this can go

Reply:
It's used for the 'use_cheriabi' check to decide between COMPAT_CHERIABI and the native ABI for purecap.  I've previously removed the 256 bit checks from it.

Reply:
Oh, yeah, that needs to stay. But the calls to (and inline implementation of) cheriabi_check_cpu_compatible are still there.